README.md

```diff
@@ -121,6 +122,9 @@ This is the path to the app's debug apk.
 ### instrumentationApk
 This is the path to the app's instrumentation apk.
 
+### additionalTestApks
```

Comment:
can you add a short sample? and that it is a map of debug apks to a list of instrumentation apks? thanks :)